optaplanner-core/src/main/java/org/optaplanner/core/api/score/Score.java

```diff
@@ -155,6 +165,22 @@
      */
     boolean isCompatibleArithmeticArgument(Score otherScore);
 
+    /**
+     * A {@link PlanningSolution} is feasible if it has no broken hard constraints
+     * and {@link #isSolutionInitialized()} is true.
+     *
+     * Simple scores ({@link SimpleScore}, {@link SimpleLongScore}, {@link SimpleBigDecimalScore}) are always feasible.
```

Comment:
not sure if it's worth adding, but "(unless their {@link #getInitScore()} isn't 0)"